forc/src/utils/plugin_descriptions.rs

```diff
@@ -0,0 +1,27 @@
+pub enum Plugin {
+    ForcFormat,
+    ForcLsp,
+    ForcExplore,
+    None,
+}
+
+pub fn plugin_name_to_enum(n: &str) -> Plugin {
+    match n {
+        "forc-fmt" => Plugin::ForcFormat,
+        "forc-explore" => Plugin::ForcExplore,
+        "forc-lsp" => Plugin::ForcLsp,
+        _ => Plugin::None,
+    }
+}
+
+pub fn plugin_description(n: &str) -> String {
+    let e = plugin_name_to_enum(n);
+    match e {
+        Plugin::ForcFormat => "Forc plugin for running the Sway code formatter.".to_string(),
+        Plugin::ForcLsp => {
+            "Forc plugin for the Sway LSP (Language Server Protocol) implementation".to_string()
+        }
+        Plugin::ForcExplore => "Forc plugin for running the Fuel Block Explorer.".to_string(),
+        Plugin::None => "Unidentified plugin".to_string(),
+    }
+}
```

Comment:
Rather than hardcoding these, could we instead add a `--description` option to each plugin or something? We have to consider the case that third-party plugins are installed for which the source code isn't in the Sway repo.

Reply:
@adlerjohn  I'm sure that's definitely possible. But the original issue doesn't mention this additional context. Could you add this context to https://github.com/FuelLabs/sway/issues/702. Obviously being newer I'm not familiar with anything relating to 3rd party stuff and the issue doesn't mention it